migration_steps/transform_casrec/transform/app/entities/finance_order/finance_order.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+from custom_errors import EmptyDataFrame
+
+from transform_data.apply_datatypes import reapply_datatypes_to_fk_cols
+from utilities.basic_data_table import get_basic_data_table
+import logging
+import os
+
+from helpers import get_mapping_dict, get_table_def
+
+log = logging.getLogger("root")
+
+
+def insert_finance_order(target_db, db_config, mapping_file):
+    chunk_size = db_config["chunk_size"]
+    offset = -chunk_size
+    chunk_no = 0
+
+    orders_query = (
+        f'SELECT "id" AS order_id, "c_order_no" FROM {db_config["target_schema"]}.cases;'
+    )
+    orders_df = pd.read_sql_query(orders_query, db_config["db_connection_string"])
+    orders_df = orders_df[["order_id", "c_order_no"]]
+
+    risk_assessment_query = (
+        f'''
+            SET datestyle = "ISO, DMY";
+            SELECT MIN("Start Date"::date) AS billing_start_date,
+            "Order No" AS c_order_no
+            FROM {db_config["source_schema"]}.risk_assessment
+            GROUP BY "Order No";
+        '''
+    )
+    risk_assessment_df = pd.read_sql_query(risk_assessment_query, db_config["db_connection_string"])
+    risk_assessment_df = risk_assessment_df[["billing_start_date", "c_order_no"]]
```

Comment:
This line is not necessary as these are the only two fields in the dataframe anyway.

Reply:
True 🤦